### ISSUE TITLE
Nested messages in JSON

### ISSUE THREAD

**Issue by ewhauser** [danielgtaylor/python-betterproto#8]
I'm getting some unexpected behavior when doing JSON serialization. Simple example:

```python
        event = Event(
            event_id=str(uuid.uuid4()),
        )

        envelope = Envelope(
            event=event,
        )

       data = envelope.to_json().encode('utf-8')
```

When I go to serialize the `envelope`, `event` is not a serialized property on the output. Is this intended behavior?

**Comment by qix** [comment]
I think this was a similar issue to what I was seeing, solved it in #10

**Comment by danielgtaylor** [comment]
Closing as this is now fixed with #10 being merged.
